apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/HHAClaimTransformerV2.java

```diff
@@ -225,6 +225,7 @@ private static ExplanationOfBenefit transformClaim(HHAClaim claimGroup) {
       // REV_CNTR_NCVRD_CHRG_AMT    => ExplanationOfBenefit.item.adjudication
       // REV_CNTR_NDC_QTY           => ExplanationOfBenefit.item.quantity
       // REV_CNTR_NDC_QTY_QLFR_CD   => ExplanationOfBenefit.modifier
+      // REV_CNTR_UNIT_CNT  => ExplanationOfBenefit.item.extension.valueQuantity
```

Comment:
minor nit: this is not lined up here or in the other places below

Reply:
done